plugins/memory/src/memory-handler.ts

```diff
@@ -106,8 +106,8 @@ class MemoryHandler implements IPackageStorageManager {
     const temporalName = `/${name}`;
 
     process.nextTick(function() {
-      fs.exists(temporalName, function(exists) {
-        if (exists) {
+      fs.stat(temporalName, function(fileError, stats) {
+        if (!fileError && stats) {
```

Comment:
Why `!fileError`? When it is set there is an error. If it is not set set (== false) there is no error.